resources/views/account/partials/layout.blade.php

```diff
@@ -15,7 +15,7 @@
         class="container"
         v-cloak
     >
-        <template v-if="$root.user?.id">
+        <template v-if="$root.loggedIn">
```

Comment:
```suggestion
        <template v-if="loggedIn">
```
Should also work, but test it